deploy/charts/rook-ceph/values.yaml

```diff
@@ -7,7 +7,7 @@ image:
   repository: rook/ceph
   # -- Image tag
   # @default -- `master`
-  tag: VERSION
```

Comment:
The helm build replaces this version when it publishes the helm chart, so I don't think we can change this.

Reply:
I just make the build change,
Lemme know if it's feel good, otherwise I can revert the cahnges

Reply:
I believe this will work fine since the makefile is updated. 
1. You're making this change so it's easier to test the local helm chart with the master tag, right?
2. Please create a new commit for this tag change. It's not just a doc improvement like the rest.

Reply:
Yes exactly, 
Added it as separate commit
Thanks